src/loader/loader.c

```diff
@@ -0,0 +1,288 @@
+#include <postgres.h>
+#include <pg_config.h>
+#include <access/xact.h>
+#include <commands/extension.h>
```

Comment:
commands/extension.h, miscadmin.h, and parser/analyze.h need to be guarded on msvc:

```
#include "../compat-msvc-enter.h"	/* To label externs in these headers correctly */
#include <commands/extension.h>
#include <miscadmin.h>
#include <parser/analyze.h>
#include "../compat-msvc-exit.h"
```

Reply:
Fixed